src/program.js

```diff
@@ -121,7 +129,9 @@ export class Program {
     argv.customPrefs = argv.pref;
 
     const runCommand = this.commands[cmd];
-
+    const configFileName = argv.config;
+    const configObject = parseConfig(argv, configFileName);
+    const newArgv = applyConfigToArgv(argv, configObject, this.defaultValues);
```

Comment:
As we discussed, remove all of this from the patch. We add it back to `program.js` in another patch. By removing it, it will allow you to focus just on the units themselves and not have to deal with failures in `test.program.js`.